x/operator/keeper/opt.go

```diff
@@ -29,6 +29,10 @@ func (k *Keeper) OptIn(
 	if isAvs, _ := k.avsKeeper.IsAVS(ctx, avsAddr); !isAvs {
 		return types.ErrNoSuchAvs.Wrapf("AVS not found %s", avsAddr)
 	}
+	// check if operator is in the whitelist
+	if _, err := k.avsKeeper.IsWhitelisted(ctx, avsAddr, operatorAddress.String()); err != nil {
```

Comment:
Only check error? Why not check if it's exist in whitelist?

Reply:
IsWhitelisted This method internally checks whether it is in a whitelisted list. If avs sets a whitelisted list, it will throw an err if opt's operator is not in this list